### ISSUE TITLE
Optional "timestamp" value missing

### ISSUE THREAD

**Issue by felipe-conde-benavides** [ogibayashi/kafka-topic-exporter#4]
Currently the following metric format is exported like this:
```
<kafka topic name>_<metric_name>{foolabel="foolabelvalue", barlabel="barlabelvalue"} <metric value>
```
Prometheus allows to send the "timestamp" value as follows:
```
metric_name [
  "{" label_name "=" `"` label_value `"` { "," label_name "=" `"` label_value `"` } [ "," ] "}"
] value [ timestamp ]
```

So, I suggest to add, **as optional**, another property into the JSON structure:
```
{"name": <metric name>,
 "value": <metric value>,
 "timestamp": <epochvalue>,
 "labels: {
    "foolabel": "foolabelvalue",
    "barlabel": "barlabelvalue"
    }
}
```
The final metric format would be:
```
<kafka topic name>_<metric_name>{foolabel="foolabelvalue", barlabel="barlabelvalue"} <metric value> [<timestamp>]
```

**Comment by cyberluisda** [comment]
+1

**Comment by felipe-conde-benavides** [comment]
I'm working on it so, once the people from "Prometheus Java Client" accept my changes on their code, I will be ready for doing a PR with the changes that I have implemented.

**Comment by ogibayashi** [comment]
Great!

**Comment by felipe-conde-benavides** [comment]
My changes in "Prometheus Java Client" have been accepted and merged, but still there isn't any release that containing them. They exists only in the `master` branch, we should wait until next library version will be released.

**Comment by felipe-conde-benavides** [comment]
The functionality about the `timestamp` has been already implemented in the version 0.0.5, so this issue can be set as _Closed_
